internal/pkg/util/uri/uri.go

```diff
@@ -53,7 +56,7 @@ func IsValid(source string) (valid bool, err error) {
 }
 
 // GetName turns a transport:ref URI into a name containing the top-level identifier
-// of the image. For example, docker://sylabs.io/lolcow returns lolcow
+// of the image. For example, docker://sylabsio/lolcow returns lolcow
```

Comment:
@DrDaveD this is not a proper hostname

Reply:
Right but it is a proper docker URI.  `apptainer pull` with the listed URI works and not with the old one.  sylabs.io does not support the docker API.